### ISSUE TITLE
If configuration files such as options.txt exist, it should be supplemented with the configuration file instead of doing nothing

### ISSUE THREAD

**Issue by GodGun968** [shedaniel/your-options-shall-be-respected#11]
Merge files, Add configurations that do not exist, instead of creating files if they do not exist

**Comment by Frontear** [comment]
+1 to this, I often create a sparse options.txt file with just my fov and mouseSensitivity changes, and with the current implementation of YOSBR, it will simply do nothing. Instead, it would be really nice if it could go through each flag and set the ones that don't exist. It would honestly be such a nice QOL improvement.

**Comment by lucasteo16** [comment]
yes, I think this mod simply won't do anything if the file exist, it should parse the information from the config and check if key values exist in the actual config during the initial launch.

just support a handful of standard configs is definately enough
`.json`, `.properties`, special file such as `option.txt` maybe `.toml` etc. If none of it exist then fallback to just check if the file exist. But `option.txt` is definitely the focus here. One problem with this is that some launcher e.g. `modrinth` they create option files by themselves sometimes controlled via their settings during initial launch, thus causing this mod not to add in preconfigured `option.txt` when modpack loaded in the first time on modrinth launcher.

Alternatively, we could define a config for `yosbr` specifying which file should be checked in a this granular way so that modpack dev can specify which config should always be parsed during initial run, this will reduce useless config parsing and be able to target specific file.

**Comment by GodGun968** [comment]
Is there no one else who will deal with this problem?
